plugins/onepad/wx_dialog/dialog.h

```diff
@@ -61,7 +61,7 @@ enum gui_buttons {
 #define BUTTONS_LENGHT 32 // numbers of buttons on the gamepad
 #define UPDATE_TIME 5
 #define DEFAULT_WIDTH 1000
-#define DEFAULT_HEIGHT 740
+#define DEFAULT_HEIGHT 820
```

Comment:
Wouldn't this negatively affect people with smaller screens?

Reply:
https://www.w3schools.com/browsers/browsers_display.asp According to this ~30% of screens will be negatively impacted. Its probably possible to get the resolution and then set the value dynamically?